### PR TITLE
Integration of CircleCi in the repo 

### PR DESCRIPTION
**Pull request by NicoFunes**
## Summary

[Change!] Describe your feature, problems you had, notes, improvements and others.

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] I have reviewed the files changed before releasing the pull request
- [x] BEFORE MERGE: I have updated the TRELLO card column before merging the pull request

## Screenshots

[Change!] Show the screenshots of the views you modified.
PullRequestImage

## TRELLO Card

[Change!] Link to the associated TRELLO card.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -39,6 +39,10 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'execjs'
```
Comment (pepeexample) [inline Gemfile]: this can be removed as we discussed

Comment (NicoFunes) [reply Gemfile]: I removed the extra gems in the Gemfile 
ImagenPullRequest2

**Comment by NicoFunes** [comment]
I completed the devise and user cards from trello.

File: log/development.log [hunk 2]
```diff
@@ -4406,3 +4406,77 @@ Processing by Rails::WelcomeController#index as HTML
 Completed 200 OK in 15ms (Views: 6.8ms | ActiveRecord: 0.0ms)
```
Comment (pepeexample) [inline log/development.log]: add this directory to gitignore

File: test/models/model_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ModelTest < ActiveSupport::TestCase
```
Comment (pepeexample) [inline test/models/model_test.rb]: this class can be deleted for the moment

File: test/models/user_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
```
Comment (pepeexample) [inline test/models/user_test.rb]: this class can be deleted for the moment

File: log/development.log (hunk 2 above)
Comment (NicoFunes) [reply log/development.log]: Directory log added to gitignore
removeLogDirectory

File: test/models/model_test.rb (hunk 3 above)
Comment (NicoFunes) [reply test/models/model_test.rb]: Class deleted 
removeTest

File: test/models/user_test.rb (hunk 4 above)
Comment (NicoFunes) [reply test/models/user_test.rb]: Class deleted 
removeTest

**Pull request merged**